proto/meta.proto

```diff
@@ -165,9 +165,15 @@ service StreamManagerService {
 // Below for cluster service.
 
 message AddWorkerNodeRequest {
+  message Property {
```

Comment:
Why we need two properties? How about merging them into one `WorkerNodeProperty`?

Reply:
Do you mean there are two `Property` proto message? Because one is used in RPC where parallelism is a int, while another is used in meta model where parallelism has been converted to concrete parallel units.